### PR TITLE
brlcad: 7.34.2 -> 7.38.2

### LINKED ISSUE #279151: Update request: brlcad 7.34.2 → 7.38.0
- Package name: brlcad  
- Latest released version: 7.38.0
- Current version on the unstable channel: 7.34.2
- Current version on the stable/release channel: 7.34.2
- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

@GaetanLepage

https://github.com/BRL-CAD/brlcad/releases/tag/rel-7-38-0

-----

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Description of changes

Changelog: https://github.com/BRL-CAD/brlcad/releases/tag/rel-7-38-2

Fixes #279151

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by GaetanLepage** [comment]
This currently fails to build:
```
[ 41%] Building CXX object code/CMakeFiles/assimp.dir/AssetLib/HMP/HMPLoader.cpp.o
[ 41%] Building CXX object code/CMakeFiles/assimp.dir/AssetLib/Irr/IRRMeshLoader.cpp.o
[ 42%] Building CXX object code/CMakeFiles/assimp.dir/AssetLib/Irr/IRRShared.cpp.o

-- stderr output is:
In file included from /build/source/src/other/ext/assetimport/include/assimp/ParsingUtils.h:54,
                 from /build/source/src/other/ext/assetimport/code/AssetLib/AMF/AMFImporter_Geometry.cpp:45:
In function 'char_t ai_tolower(char_t) [with char_t = const char*]',
    inlined from 'void Assimp::AMFImporter::ParseNode_Coordinates(Assimp::XmlNode&)' at /build/source/src/other/ext/assetimport/code/AssetLib/AMF/AMFImporter_Geometry.cpp:168:75:
/build/source/src/other/ext/assetimport/include/assimp/StringUtils.h:220:53: error: offset '32' outside bounds of constant string [-Werror=array-bounds]
  220 |     return (in >= (char_t)'A' && in <= (char_t)'Z') ? (char_t)(in + 0x20) : in;
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[5]: *** [code/CMakeFiles/assimp.dir/build.make:1042: code/CMakeFiles/assimp.dir/AssetLib/AMF/AMFImporter_Geometry.cpp.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[4]: *** [CMakeFiles/Makefile2:169: code/CMakeFiles/assimp.dir/all] Error 2
make[3]: *** [Makefile:136: all] Error 2

CMake Error at /build/source/build/src/other/ext/ASSETIMPORT_BLD-prefix/src/ASSETIMPORT_BLD-stamp/ASSETIMPORT_BLD-build-Release.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [src/other/ext/CMakeFiles/ASSETIMPORT_BLD.dir/build.make:86: src/other/ext/ASSETIMPORT_BLD-prefix/src/ASSETIMPORT_BLD-stamp/ASSETIMPORT_BLD-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:13733: src/other/ext/CMakeFiles/ASSETIMPORT_BLD.dir/all] Error 2
```

**Comment by GaetanLepage** [comment]
> This currently fails to build:
> 
> ```
> [ 41%] Building CXX object code/CMakeFiles/assimp.dir/AssetLib/HMP/HMPLoader.cpp.o
> [ 41%] Building CXX object code/CMakeFiles/assimp.dir/AssetLib/Irr/IRRMeshLoader.cpp.o
> [ 42%] Building CXX object code/CMakeFiles/assimp.dir/AssetLib/Irr/IRRShared.cpp.o
> 
> -- stderr output is:
> In file included from /build/source/src/other/ext/assetimport/include/assimp/ParsingUtils.h:54,
>                  from /build/source/src/other/ext/assetimport/code/AssetLib/AMF/AMFImporter_Geometry.cpp:45:
> In function 'char_t ai_tolower(char_t) [with char_t = const char*]',
>     inlined from 'void Assimp::AMFImporter::ParseNode_Coordinates(Assimp::XmlNode&)' at /build/source/src/other/ext/assetimport/code/AssetLib/AMF/AMFImporter_Geometry.cpp:168:75:
> /build/source/src/other/ext/assetimport/include/assimp/StringUtils.h:220:53: error: offset '32' outside bounds of constant string [-Werror=array-bounds]
>   220 |     return (in >= (char_t)'A' && in <= (char_t)'Z') ? (char_t)(in + 0x20) : in;
>       |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
> cc1plus: all warnings being treated as errors
> make[5]: *** [code/CMakeFiles/assimp.dir/build.make:1042: code/CMakeFiles/assimp.dir/AssetLib/AMF/AMFImporter_Geometry.cpp.o] Error 1
> make[5]: *** Waiting for unfinished jobs....
> make[4]: *** [CMakeFiles/Makefile2:169: code/CMakeFiles/assimp.dir/all] Error 2
> make[3]: *** [Makefile:136: all] Error 2
> 
> CMake Error at /build/source/build/src/other/ext/ASSETIMPORT_BLD-prefix/src/ASSETIMPORT_BLD-stamp/ASSETIMPORT_BLD-build-Release.cmake:47 (message):
>   Stopping after outputting logs.
> 
> 
> make[2]: *** [src/other/ext/CMakeFiles/ASSETIMPORT_BLD.dir/build.make:86: src/other/ext/ASSETIMPORT_BLD-prefix/src/ASSETIMPORT_BLD-stamp/ASSETIMPORT_BLD-build] Error 1
> make[1]: *** [CMakeFiles/Makefile2:13733: src/other/ext/CMakeFiles/ASSETIMPORT_BLD.dir/all] Error 2
> ```

This is now working.
The last remaining error occurs at the very end of the building process (after all the `shrinking`):
```
checking for references to /build/ in /nix/store/rq2hjvfgq2nvh5zxch51ij34rqqdpark-brlcad-7.38.0...
RPATH of binary /nix/store/rq2hjvfgq2nvh5zxch51ij34rqqdpark-brlcad-7.38.0/bin/tclsh contains a forbidden reference to /build/
```

File: pkgs/applications/graphics/brlcad/default.nix [hunk 1]
```diff
@@ -11,17 +12,18 @@
...
-    repo = pname;
+    repo = "brlcad";
     rev = "refs/tags/rel-${lib.replaceStrings [ "." ] [ "-" ] version}";
-    hash = "sha256-oafu255xElEIk8p4yvNyR2maykUfxQui/L5MkicA+JA=";
+    hash = "sha256-Q6+PewMuEqsysq2/gS1VoMJfTBU7U/4tDm0XgNch/pI=";
   };
 
   nativeBuildInputs = [
     cmake
+    patchelf  # TODO remove
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/brlcad/default.nix:26]:
```suggestion
```
that's part of stdenv

File: pkgs/applications/graphics/brlcad/default.nix [hunk 2]
```diff
@@ -36,9 +38,21 @@ stdenv.mkDerivation rec {
...
   ];
 
+  env.NIX_CFLAGS_COMPILE = toString [
+    # Needed with GCC 12
+    "-Wno-error=array-bounds"
+  ];
+
+  postInstall = ''
+    echo "----------------------------------------------------"
+    patchelf --print-rpath "$out/bin/tclsh"
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/brlcad/default.nix:48]:
```
----------------------------------------------------
/build/source/build/lib::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::/nix/store/rq2hjvfgq2nvh5zxch51ij34rqqdpark-brlcad-7.38.0/lib:/nix/store/4znq11s8j9d29kj2l4qivl5pyhbbzy8q-zlib-1.3/lib:/nix/store/7jiqcrg061xi5clniy7z5pvkc4jiaqav-glibc-2.38-27/lib:/nix/store/np3cndfk53miqg2cilv7vfdxckga665h-gcc-13.2.0-lib/lib
----------------------------------------------------
```

not sure what is there

File: pkgs/applications/graphics/brlcad/default.nix (hunk 1 above)
Comment (GaetanLepage) [reply pkgs/applications/graphics/brlcad/default.nix]: Removed

File: pkgs/applications/graphics/brlcad/default.nix (hunk 2 above)
Comment (GaetanLepage) [reply pkgs/applications/graphics/brlcad/default.nix:46]: After a never ending `bisect`, I ended up finding the culprit:
https://github.com/BRL-CAD/brlcad/commit/fbdbf042b2db4c7d46839a17bbf4985cdb81f0ae
@SuperSandro2000 do you know what to do in this case ?

**Review by SuperSandro2000 — APPROVED** [review]
diff LGTM

**Comment by GaetanLepage** [comment]
> diff LGTM

Yes, but it still doesn't build

**Comment by GaetanLepage** [comment]
The `mged` binary runs fine on my system.

**Review by JohnRTitor — APPROVED** [review]
Result of `nixpkgs-review pr 279705` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>brlcad</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Facing this on `aarch64-linux`:
```
/build/source/src/other/openNURBS/opennurbs_system.h:704:2: error: #error Either ON_LITTLE_ENDIAN or ON_BIG_ENDIAN must be defined.
  704 | #error Either ON_LITTLE_ENDIAN or ON_BIG_ENDIAN must be defined.
      |  ^~~~~
```

**Comment by JohnRTitor** [comment]
Arm is a little endian architecture. I think that macro can be set to 1, via a patch?

**Comment by GaetanLepage** [comment]
> Arm is a little endian architecture. I think that macro can be set to 1, via a patch?

Probably. Not sure how to do it though...
Seems related to https://github.com/BRL-CAD/brlcad/discussions/102

**Comment by drupol** [comment]
What's the status here ?

**Comment by JohnRTitor** [comment]
Can you mark it broken or aarch64-linux? I will try to fix it via a patch later.

**Comment by GaetanLepage** [comment]
After all, it seems that it is not building on `x86_64` either:

<details>

```
[ 94%] Built target appslib
[ 94%] Built target gcore
[ 94%] Linking CXX shared library libgdal.so

-- stderr output is:
...skipping to end...
            ^
/build/source/src/other/ext/gdal/alg/internal_libqhull/io_r.c: In function 'gdal_qh_printcentrum':
/build/source/src/other/ext/gdal/alg/internal_libqhull/io_r.c:1662:9: note: 'xaxis' declared here
 1662 |   realT xaxis[4], yaxis[4], normal[4], dist;
      |         ^~~~~
In file included from /build/source/src/other/ext/gdal/port/cpl_minixml.h:33,
                 from /build/source/src/other/ext/gdal/gcore/gdal_pam.h:35,
                 from /build/source/src/other/ext/gdal/frmts/ilwis/ilwisdataset.h:32,
                 from /build/source/src/other/ext/gdal/frmts/ilwis/ilwisdataset.cpp:30:
/build/source/src/other/ext/gdal/frmts/ilwis/ilwisdataset.cpp: In member function 'void GDAL::IniFile::Load()':
/build/source/src/other/ext/gdal/port/cpl_port.h:1135:25: warning: attributes at the beginning of statement are ignored [-Wattributes]
 1135 | #define CPL_FALLTHROUGH [[clang::fallthrough]];
      |                         ^~~~~~~~~~~~~~~~~~~~~~
/build/source/src/other/ext/gdal/frmts/ilwis/ilwisdataset.cpp:207:17: note: in expansion of macro 'CPL_FALLTHROUGH'
  207 |                 CPL_FALLTHROUGH
      |                 ^~~~~~~~~~~~~~~
/build/source/src/other/ext/gdal/frmts/ilwis/ilwisdataset.cpp:206:28: warning: this statement may fall through [-Wimplicit-fallthrough=]
  206 |                 s = GetLine(filIni);  // fall through (no break)
      |                     ~~~~~~~^~~~~~~~
/build/source/src/other/ext/gdal/frmts/ilwis/ilwisdataset.cpp:208:13: note: here
  208 |             case FindKey:
      |             ^~~~
In file included from /nix/store/zc0nsv23pakbafngjy32kvhfzb16as43-gcc-13.3.0/include/c++/13.3.0/bits/hashtable.h:35,
                 from /nix/store/zc0nsv23pakbafngjy32kvhfzb16as43-gcc-13.3.0/include/c++/13.3.0/bits/unordered_set.h:33,
                 from /nix/store/zc0nsv23pakbafngjy32kvhfzb16as43-gcc-13.3.0/include/c++/13.3.0/unordered_set:41,
                 from /build/source/src/other/ext/gdal/frmts/vrt/vrtmultidim.cpp:35:
In member function 'std::__detail::_Hash_code_base<_Key, _Value, _ExtractKey, _Hash, _RangeHash, _Unused, __cache_hash_code>::__hash_code std::__detail::_Hash_code_base<_Key, _Value, _ExtractKey, _Hash, _RangeHash, _Unused, __cache_hash_code>::_M_hash_code(const _Key&) const [with _Key = const void*; _Value = const void*; _ExtractKey = std::__detail::_Identity; _Hash = std>
    inlined from 'std::size_t std::__detail::_Hash_code_base<_Key, _Value, _ExtractKey, _Hash, _RangeHash, _Unused, __cache_hash_code>::_M_bucket_index(const std::__detail::_Hash_node_value<_Value, false>&, std::size_t) const [with _Key = const void*; _Value = const void*; _ExtractKey = std::__detail::_Identity; _Hash = std::hash<const void*>; _RangeHash = std::__detail::_>
    inlined from 'std::_Hashtable<_Key, _Value, _Alloc, _ExtractKey, _Equal, _Hash, _RangeHash, _Unused, _RehashPolicy, _Traits>::size_type std::_Hashtable<_Key, _Value, _Alloc, _ExtractKey, _Equal, _Hash, _RangeHash, _Unused, _RehashPolicy, _Traits>::_M_bucket_index(const __node_value_type&) const [with _Key = const void*; _Value = const void*; _Alloc = std::allocator<con>
    inlined from 'std::_Hashtable<_Key, _Value, _Alloc, _ExtractKey, _Equal, _Hash, _RangeHash, _Unused, _RehashPolicy, _Traits>::iterator std::_Hashtable<_Key, _Value, _Alloc, _ExtractKey, _Equal, _Hash, _RangeHash, _Unused, _RehashPolicy, _Traits>::erase(const_iterator) [with _Key = const void*; _Value = const void*; _Alloc = std::allocator<const void*>; _ExtractKey = st>
    inlined from 'std::_Hashtable<_Key, _Value, _Alloc, _ExtractKey, _Equal, _Hash, _RangeHash, _Unused, _RehashPolicy, _Traits>::iterator std::_Hashtable<_Key, _Value, _Alloc, _ExtractKey, _Equal, _Hash, _RangeHash, _Unused, _RehashPolicy, _Traits>::erase(iterator) [with _Key = const void*; _Value = const void*; _Alloc = std::allocator<const void*>; _ExtractKey = std::__d>
    inlined from 'std::unordered_set<_Value, _Hash, _Pred, _Alloc>::iterator std::unordered_set<_Value, _Hash, _Pred, _Alloc>::erase(iterator) [with _Value = const void*; _Hash = std::hash<const void*>; _Pred = std::equal_to<const void*>; _Alloc = std::allocator<const void*>]' at /nix/store/zc0nsv23pakbafngjy32kvhfzb16as43-gcc-13.3.0/include/c++/13.3.0/bits/unordered_set.h>
    inlined from 'VRTMDArraySourceFromArray::~VRTMDArraySourceFromArray()' at /build/source/src/other/ext/gdal/frmts/vrt/vrtmultidim.cpp:1852:27:
/nix/store/zc0nsv23pakbafngjy32kvhfzb16as43-gcc-13.3.0/include/c++/13.3.0/bits/hashtable_policy.h:1310:25: warning: potential null pointer dereference [-Wnull-dereference]
 1310 |         return _M_hash()(__k);
      |                ~~~~~~~~~^~~~~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp: In function 'void AllocSprintf(char**, size_t*, const char*, __va_list_tag*)':
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:36:27: warning: attributes at the beginning of statement are ignored [-Wattributes]
   36 | #  define CPL_FALLTHROUGH [[clang::fallthrough]];
      |                           ^~~~~~~~~~~~~~~~~~~~~~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:245:16: note: in expansion of macro 'CPL_FALLTHROUGH'
  245 |                CPL_FALLTHROUGH
      |                ^~~~~~~~~~~~~~~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:36:27: warning: attributes at the beginning of statement are ignored [-Wattributes]
   36 | #  define CPL_FALLTHROUGH [[clang::fallthrough]];
      |                           ^~~~~~~~~~~~~~~~~~~~~~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:265:16: note: in expansion of macro 'CPL_FALLTHROUGH'
  265 |                CPL_FALLTHROUGH
      |                ^~~~~~~~~~~~~~~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:235:16: warning: this statement may fall through [-Wimplicit-fallthrough=]
  235 |                if ((p1 - p) == 1) {
      |                ^~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:246:13: note: here
  246 |             case 'S':
      |             ^~~~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:247:16: warning: this statement may fall through [-Wimplicit-fallthrough=]
  247 |                if ((p1 - p) == 1) {
      |                ^~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:266:13: note: here
  266 |             default:
      |             ^~~~~~~
/build/source/src/other/ext/gdal/frmts/grib/degrib/degrib/myerror.cpp:269:24: warning: 'char* __builtin___strncpy_chk(char*, const char*, long unsigned int, long unsigned int)' specified bound <unknown> equals destination size [-Wstringop-truncation]
  269 |                strncpy (buffer + ipos, p + 1, p1 - p);
      |                        ^
In file included from /nix/store/4vgk1rlzdqjnpjicb5qcxjcd4spi7wyw-glibc-2.39-52-dev/include/string.h:548,
                 from /build/source/src/other/ext/gdal/port/cpl_port.h:143,
                 from /build/source/src/other/ext/gdal/port/cpl_conv.h:35,
                 from /build/source/src/other/ext/gdal/ogr/ogrsf_frmts/avc/avc.h:118,
                 from /build/source/src/other/ext/gdal/ogr/ogrsf_frmts/avc/avc_e00gen.cpp:93:
In function 'char* strncpy(char*, const char*, size_t)',
    inlined from 'const char* AVCE00GenTableRec(AVCE00GenInfo*, int, AVCFieldInfo*, AVCField*, GBool)' at /build/source/src/other/ext/gdal/ogr/ogrsf_frmts/avc/avc_e00gen.cpp:1411:16:
/nix/store/4vgk1rlzdqjnpjicb5qcxjcd4spi7wyw-glibc-2.39-52-dev/include/bits/string_fortified.h:95:34: warning: 'char* __builtin_strncpy(char*, const char*, long unsigned int)' accessing 1 or more bytes at offsets 0 and [-2147483648, 2147483647] may overlap up to 18446744071562067968 bytes at offset [0, 2147483647] [-Wrestrict]
   95 |   return __builtin___strncpy_chk (__dest, __src, __len,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
   96 |                                   __glibc_objsize (__dest));
      |                                   ~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/src/other/ext/gdal/ogr/ogrsf_frmts/sqlite/ogrsqlitevirtualogr.cpp:1828:1: warning: missing initializer for member 'sqlite3_module::xIntegrity' [-Wmissing-field-initializers]
 1828 | };
      | ^
/nix/store/wwfrj9kvfi14xclc38qfwm71ah6aawdh-binutils-2.41/bin/ld: final link failed: No space left on device
collect2: error: ld returned 1 exit status
make[5]: *** [CMakeFiles/GDAL.dir/build.make:2232: libgdal.so.32.3.6.3] Error 1
make[4]: *** [CMakeFiles/Makefile2:3597: CMakeFiles/GDAL.dir/all] Error 2
make[3]: *** [Makefile:146: all] Error 2

CMake Error at /build/source/build/src/other/ext/GDAL_BLD-prefix/src/GDAL_BLD-stamp/GDAL_BLD-build-Release.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [src/other/ext/CMakeFiles/GDAL_BLD.dir/build.make:86: src/other/ext/GDAL_BLD-prefix/src/GDAL_BLD-stamp/GDAL_BLD-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:13451: src/other/ext/CMakeFiles/GDAL_BLD.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

</details>

**Comment by GaetanLepage** [comment]
Actually, it does build. I'll mark the build as broken on ARM.

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 279705` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>brlcad</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 279705` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>brlcad</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 279705` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>brlcad</li>
  </ul>
</details>

**Pull request merged**